proofread: sync_accounts and upgrade (#2511)
juicedata/juicefs: docs/en/guide/quota.md

---
# Storage Quota

Since JuiceFS v0.14.2, JuiceFS have supported file system level storage quotas. This feature allows users to:

- Limit the total available capacity of the file system
- Limit the total inodes of the file system

:::tip
The storage quota settings are stored in the metadata engine for all mount points to read, and the client of each mount point will also cache its own used capacity and inodes and synchronize them with the metadata engine once per second. Meanwhile the client will read the latest usage value from the metadata engine every 10 seconds to synchronize the usage information among each mount point, but this information synchronization mechanism cannot guarantee that the usage data is counted accurately.
:::

## View file system information

> In a Linux environment, for example, the default capacity of a JuiceFS type file system is identified as `1.0P` by using the `df` command that comes with the system.

Reviewer: Safe to assume that Linux users are familiar with df.

For Linux, the default capacity of a JuiceFS type file system is identified as `1.0P` by using the `df` command.

> In a Linux environment, for example, the default capacity of a JuiceFS type file system is identified as `1.0P` by using the `df` command that comes with the system.

Author: ```suggestion
For Linux, the default capacity of a JuiceFS type file system is identified as `1.0P` by using the `df` command.
```

```shell
$ df -Th | grep juicefs
JuiceFS:ujfs   fuse.juicefs  1.0P  682M  1.0P    1% /mnt
```

:::note

> The capacity of underlying object storage is usually unlimited; namely, JuiceFS storage is unlimited. Therefore, the displayed capacity is just an estimate rather than the actual capacity.

Reviewer: reduce word occurrence for "capacity":

Therefore, the displayed capacity is just an estimate rather than an actual storage limit.

Reviewer: namely => i.e.

> The capacity of underlying object storage is usually unlimited; namely, JuiceFS storage is unlimited. Therefore, the displayed capacity is just an estimate rather than the actual capacity.

Author: ```suggestion
The capacity of underlying object storage is usually unlimited, i.e., JuiceFS storage is unlimited. Therefore, the displayed capacity is just an estimate rather than the actual storage limit.
```

> JuiceFS implements support for the POSIX interface through FUSE, because the underlying object storage is usually of infinitely scalable capacity. Therefore, the identification capacity is just an estimate (which also means unlimited) and not the actual capacity, which changes dynamically with the actual usage.

Reviewer: better off without the FUSE recap, something like:

We don't usually enforce storage limit on the underlying object storage, therefore, the displayed capacity is just an estimate, which means JuiceFS storage is unlimited.

I'd avoid using the phrase "identification capacity", search shows it's not usually associated with computer file system / storage.

Author: ```suggestion
The capacity of underlying object storage is usually unlimited; namely, JuiceFS storage is unlimited. Therefore, the displayed capacity is just an estimate rather than the actual capacity.
```
I tried to avoid "We" in the sentence, so the structure was changed a bit. Hope it did not change the meaning of it?
